tests/test_stlcontainers.py

```diff
@@ -656,8 +656,6 @@ def test_map_str_vector_double():
     uismap = isinstance([1.0, -65.5555, 1.0, -65.5555], Mapping) 
     m['Aha'] = [42.42, 18, 42.42, 18]
     m['Take'] = [1.0, -65.5555, 1.0, -65.5555]
-    import pprint
```

Comment:
(I understood why you deleted them... just wanted to make sure they weren't going to be missed.)

Sounds good to me. Merging!